### PR TITLE
Move compatibilities to main mod and conditionally load

### PR DESCRIPTION
**Pull request by Mike-MF**
**When merged this pull request will:**
- Moves CSLA, GM, SOG & WS compats into main.
- Utilises `skipWhenMissingDependencies`
- Uses `QPATHTOF` for WRP file.

### CODE REVIEW

File: addons/compat_csla/config.cpp [hunk 1]
```diff
@@ -11,11 +11,12 @@ class CfgPatches {
         authors[] = {"CSLA Studio"};
         url = ECSTRING(main,URL);
         VERSION_CONFIG;
+        skipWhenMissingDependencies = 1;
```
Comment (PabstMirror) [inline addons/compat_csla/config.cpp]: I don't think we even need these
can't think of a case where someone will have the map_compat but not the base acre

Comment (Mike-MF) [reply addons/compat_csla/config.cpp]: Ah, you got me just before I updated requiredAddons.

Comment (PabstMirror) [reply addons/compat_csla/config.cpp]: we don't really need the mod in requiredAddons
we could just leave it like before and the compats just always load and it shouldn't hurt anything

**Comment by PabstMirror** [comment]
I think we want to drop the meta.cpps?

File: addons/compat_csla/config.cpp (hunk 1 above)
Comment (jonpas) [reply addons/compat_csla/config.cpp]: Some compats do more than just add an entry in `CfgAcreWorlds`, like SOG's radio replacement (don't want to run that function if SOG isn't even loaded) and the newly coming SPE, as well as improved compat for SOG currently in PR.

**Review by jonpas — APPROVED** [review]
Renaming shouldn't break anything unless someone required our compatibilities, which I highly doubt. If anyone knows of any, let us know.

File: addons/compat_csla/config.cpp (hunk 1 above)
Comment (PabstMirror) [reply addons/compat_csla/config.cpp]: my concern is that we have to wait for 2.14 to merge this PR
not sure if we want to wait for that to get SPE compat

we could use a `has_include` setup to get these merged now

Comment (jonpas) [reply addons/compat_csla/config.cpp]: Nah, we can wait a few weeks. Previous compats have come much later.

**Review by jonpas — APPROVED** [review]

**Pull request merged**